test/integration/transformation_filter_integration_test.cc

```diff
@@ -220,7 +220,7 @@ class TransformationFilterIntegrationTest
         Extensions::HttpFilters::SoloHttpFilterNames::get().Transformation);
     filter.mutable_typed_config()->PackFrom(filter_config);
 
-    return MessageUtil::getJsonStringFromMessage(filter);
+    return MessageUtil::getJsonStringFromMessageOrDie(filter);
```

Comment:
this utility was changed in upstream envoy, `getJsonStringFromMessageOrDie()` has the return type `getJsonStringFromMessage()` used to have